### DISCUSSION TITLE
Not  create a repo

### DISCUSSION THREAD

**Discussion by DenysBzenko** [community/community#106393] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/119534908/02e60b88-dbbe-4489-baf1-14113a8430aa)
I  cannnot create a repo in classroom in github

**Comment by Bardin08** [comment]
@martinwoodward @dipree Sorry if these are invalid tags. Please help to address this issue.

![image](https://github.com/community/community/assets/67170413/61766d2e-e849-43c3-921c-4b1fe52fa74c)
